kibbeh/svgr.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  typescript: true,
+  replaceAttrValues: { "#000": "currentColor" },
```

Comment:
why did you change from `#fff` to `#000`?

Reply:
mistake, changed to `#fff`